WordPress/Classes/ViewRelated/Reader/Controllers/ReaderLoggedOutViewController.swift

```diff
@@ -0,0 +1,36 @@
+import UIKit
+import SwiftUI
+import WordPressUI
+
+final class ReaderLoggedOutViewController: UIViewController {
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        view.backgroundColor = .systemBackground
+
+        let stateView = UIHostingView(view: EmptyStateView {
```

Comment:
Any reason for not using `UIHostingViewController`, which may require less set up code?